code/parachain/runtime/composable/src/gates.rs

```diff
@@ -0,0 +1,114 @@
+use crate::{
+	prelude::*, weights, Balances, ReleaseCommittee, Runtime, RuntimeCall, RuntimeEvent,
+	RuntimeOrigin,
+};
+use common::{
+	fees::NativeExistentialDeposit, governance::native::ReleaseCollective, AccountId,
+	MaxStringSize, HOURS,
+};
+use composable_traits::account_proxy::ProxyType;
+use cumulus_primitives_core::relay_chain::BlakeTwo256;
+use frame_support::traits::{Contains, EitherOfDiverse, InstanceFilter};
+use frame_system::EnsureRoot;
+
+impl InstanceFilter<RuntimeCall> for ProxyType {
+	fn filter(&self, c: &RuntimeCall) -> bool {
+		match self {
+			ProxyType::Any => true,
+			ProxyType::Governance => matches!(
+				c,
+				RuntimeCall::Democracy(..) |
+					RuntimeCall::Council(..) |
+					//RuntimeCall::TechnicalCommittee(..) |
```

Comment:
why is the technical committee commented?

Reply:
will fix